appengine-java11/kotlin-ktor/pom.xml

```diff
@@ -0,0 +1,162 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.example.appengine</groupId>
+    <artifactId>kotlin-ktor</artifactId>
+    <version>0.0.1</version>
+
+    <!--
+    The parent pom defines common style checks and testing strategies for our samples.
+    Removing or replacing it should not affect the execution of the samples in anyway.
+    -->
+    <parent>
+        <groupId>com.google.cloud.samples</groupId>
+        <artifactId>shared-configuration</artifactId>
+        <version>1.0.11</version>
+    </parent>
+
+    <properties>
+        <kotlin.code.style>official</kotlin.code.style>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <kotlin.compiler.incremental>true</kotlin.compiler.incremental>
+        <main.class>io.ktor.server.netty.EngineMain</main.class>
+        <maven.compiler.target>11</maven.compiler.target>
+        <maven.compiler.source>11</maven.compiler.source>
+    </properties>
+
+    <repositories>
+        <repository>
```

Comment:
Do you really need to use these repos?

If your going to use an alternate repo, consider ours: http://maven-central.storage.googleapis.com/index.html
(unless we don't have the artifacts your looking for)

Reply:
These are the most up to date repos. Ktor doesn't keep their mvn central repo as updated.